.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+<!--
+A clear and concise description of what the problem is. Eg. "I'm always frustrated when [...]"
+-->
+
+**Describe the solution you'd like**
+<!--
+A clear and concise description of what you want to happen.
+-->
```

Comment:
What do you think about encouraging users to provide (pseudo-)code, for API changes they would like to see?

Reply:
I'll update the description to say that (pseudo-)code is welcome. But users who would not even be sure where to begin should not be discouraged either.